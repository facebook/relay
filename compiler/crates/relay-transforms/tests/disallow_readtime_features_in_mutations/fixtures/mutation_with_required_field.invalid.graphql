# expected-to-throw
mutation MyMutation {
  setName(name: "Alice") {
    name @required(action: THROW)
  }
}

# %extensions%
