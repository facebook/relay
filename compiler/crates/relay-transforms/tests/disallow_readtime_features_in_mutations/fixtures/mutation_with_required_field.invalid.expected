==================================== INPUT ====================================
# expected-to-throw
mutation MyMutation {
  setName(name: "Alice") {
    name @required(action: THROW)
  }
}

# %extensions%
==================================== ERROR ====================================
✖︎ Unexpected `@required(action: THROW)` directive in mutation response. The use of `@required(action: THROW)` is not supported in mutations.

  mutation_with_required_field.invalid.graphql:4:10
    3 │   setName(name: "Alice") {
    4 │     name @required(action: THROW)
      │          ^^^^^^^^^
    5 │   }
